### ISSUE TITLE
Checkbox Group with buttons

### ISSUE THREAD

**Issue by crudo** [ant-design/ant-design#31870]
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
Usecase is set of toggle-able properties of some sort.
Imagine wysiwyg editor bar [ B | I | U | S ] where any, some or all of the options can be selected.

![image](https://user-images.githubusercontent.com/67999/130331121-09662baf-0ef6-4d98-b663-6352e6620be3.png)

Basically it is the same as `Radio Group` with _optionType_ equal to _button_ but with ability to select multiple options.

### What does the proposed API look like?
Add `optionType` (_default_  or _button_) the same as Radio Group has.

**Comment by afc163** [comment]
Duplicate of https://github.com/ant-design/ant-design/issues/24540

**Comment by crudo** [comment]
@afc163 https://github.com/ant-design/ant-design/issues/24540#issuecomment-635238533

I do not see CheckCard in the API anywhere.

Can you provide a link?
